lib/runtime/wasmtime/imports.go

```diff
@@ -283,6 +283,83 @@ func ext_offchain_index_set_version_1(c *wasmtime.Caller, a, b int64) {
 	logger.Trace("[ext_offchain_index_set_version_1] executing...")
 }
 
+// ImportNodeRuntimeLinker adds the imports for the v0.8 runtime to linker
+func ImportNodeRuntimeLinker(store *wasmtime.Store, linker *wasmtime.Linker) error {
```

Comment:
could you update this to be named `ImportsNodeRuntime` and delete the below function? same for all the other linker functions, they can replace the existing import functions